app/services/search/postgres/user.rb

```diff
@@ -0,0 +1,74 @@
+module Search
+  module Postgres
+    class User
+      ATTRIBUTES = %i[
+        id
+        name
+        profile_image
+        username
+      ].freeze
+
+      DEFAULT_PER_PAGE = 60
+      MAX_PER_PAGE = 100
+
+      # User.search_sore used to take employer related fields into account, but they have since been moved to profile
+      # and removed from fields that are searched against.
+      HOTNESS_SCORE_ORDER = Arel.sql(%{
+        (((articles_count + comments_count + reactions_count + badge_achievements_count) * 10) * reputation_modifier)
+        DESC
+      }.squish).freeze
```

Comment:
I didn't bother to add a clause for `hotness_score ASC` because we never actually trigger that